crates/rspack_core/src/module_graph/mod.rs

```diff
@@ -282,10 +293,7 @@ impl ModuleGraph {
     let mut new_connection = old_con.clone();
     new_connection.id = ConnectionId::new();
     new_connection.original_module_identifier = original_module_identifier;
-    new_connection.module_identifier = module_identifier;
-    self
```

Comment:
why not add new connection to connections? the module.graph connections should save all of connections.

Reply:
Fixed